spec/models/post_spec.rb

```diff
@@ -1,41 +1,54 @@
 require 'rails_helper'
 
 RSpec.describe Post, type: :model do
-  subject { described_class.new(title: 'title', text: 'text') }
+  before :each do
+    @user = User.create(name: 'Abdu', photo: 'https://unsplash.com/photos/F_-0BxGuVvo', bio: 'Teacher from Yemen.',
+                        posts_counter: 0)
+    @post = Post.create(author: @user, title: 'Harru', text: 'text', comments_counter: 0,
+                        likes_counter: 0)
+  end
+
+  context '#most_recent_posts' do
+    it 'should show last three posts for user' do
+      comment1 = Comment.create(user: @user, post: @post, text: 'Hello Abdu....')
+      comment2 = Comment.create(user: @user, post: @post, text: 'Hello Abdu....')
+      comment3 = Comment.create(user: @user, post: @post, text: 'Hello Abdu....')
+      comment4 = Comment.create(user: @user, post: @post, text: 'Hello Abdu....')
+      comment5 = Comment.create(user: @user, post: @post, text: 'Hello Abdu....')
+      comment6 = Comment.create(user: @user, post: @post, text: 'Hello Abdu....')
+
+      expected_posts = [comment1, comment2, comment3, comment4, comment5, comment6]
```

Comment:
Well done by testing this functionality but please make sur to use valid test and variable names as the task here is about comments not posts  #most_recent_posts , expected_posts  refers to posts not comments

Reply:
Thank you